auth/build.gradle

```diff
@@ -34,9 +34,14 @@ android {
 
 dependencies {
     implementation project(path: ':core')
+    implementation project(path: ':metrics')
```

Comment:
this means the auth module will become dependent on the metrics module. I thought we want to avoid direct dependencies between each service modules?

Reply:
Yes, i'm also not 100% happy with this fact. Moving the interface to core might be an option because all modules depend on core anyway. But then core would have to keep track of the registered metric providers.

Reply:
So it might be worth it to have a utility library auth-metrics which provides auth metrics for auth.  
What metrics from the client side for auth are we wanting to capture?